### PR TITLE
DDF-3326 Updated FIPS 10-4 to ISO 3166-1 mapping table to correct sev…

### PR DESCRIPTION
**Pull request by beyelerb**
…eral entries

#### What does this PR do?
Updates several mappings that were incorrect in the FIPS 10-4 to ISO 3166-1 table
#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@roelens8 @jaymcnallie 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@clockard
@jlcsmith
#### How should this be tested? (List steps with links to updated documentation)
View the mappings and verify correctness against the following websites:
[ISO 3166-1](https://www.iso.org/iso-3166-country-codes.html)
[List of FIPS Country Codes](https://en.wikipedia.org/wiki/List_of_FIPS_country_codes#V)
[FIPS 10-4 Codes and History](http://efele.net/maps/fips-10/data/)
#### Any background context you want to provide?
Looks like most of these were fat-fingered, though some have changed over time
#### What are the relevant tickets?
[Mapping file for country codes from FIPS 10-4 to ISO_31661-1 contains an invalid country code.](https://codice.atlassian.net/browse/DDF-3326)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by beyelerb** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by beyelerb** [comment]
Killed the CI build, rebased after the branch, ready to start again.

**Comment by clockard** [comment]
build now

File: distribution/ddf-common/src/main/resources/etc/fipsToIso.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # FIPS 10-4 to ISO 3166-1 alpha 3 country codes from http://www.opengeocode.org/
```
Comment (roelens8) [inline distribution/ddf-common/src/main/resources/etc/fipsToIso.properties]: Should we remove the reference to http://www.opengeocode.org/? This website doesn't even seem to work anymore.

**Pull request merged**